### PR TITLE
Removed hash iptables rules

### PR DESCRIPTION
**Pull request by baude**
remove hashed networks chains and leave no turds behind.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
only @Luap99 should merge this

File: src/firewall/varktables/types.rs [hunk 1]
```diff
@@ -335,18 +329,23 @@ pub fn get_port_forwarding_chains<'a>(
...
+    //
+    // let netavark_hashed_network_chain_name = CONTAINER_CHAIN.to_string() + &pfwd.network_hash_name;
+    //
+    // postrouting.build_rule(VarkRule::new(
+    //     format!(
+    //         "-j {} -s {} {}",
+    //         netavark_hashed_network_chain_name, container_ip, comment_network_cid
+    //     ),
+    //     None,
+    // ));
```
Comment (Luap99) [inline src/firewall/varktables/types.rs:348]: remove this

**Review by Luap99 — COMMENTED** [review]
LGTM, tested manually and CI is green
@mheon PTAL

**Pull request merged**